bidwire/notifiers/notifier_utils.py

```diff
@@ -80,4 +80,20 @@ def make_doc_item_body(document):
     with tag('strong'):
         with tag('a', href=document.get_url()):
             text(document.title)
+    if document.description:
+        with tag('p'):
+            # it's already in HTML
+            doc.asis(document.description)
     return doc.getvalue()
+
+
+def make_notice_item_body(notice):
+    doc, tag, text = Doc().tagtext()
+    with tag('strong'):
+        with tag('a', href=notice.href):
+            text(notice.title)
+        if notice.description:
+            text(notice.description)
+        text(notice.start.strftime('%c') + ' - ' + notice.end.strftime('%c'))
+    text('<br/>'.join((notice.premise, notice.location, notice.city + ', ' + notice.state + ' ' + notice.postcode)))
```

Comment:
Please conform to PEP8 style guidelines. This line is too long. You can use the `autopep8` tool to do this for you:
```
autopep8 --in-place <filename>
```